src/python/turicreate/toolkits/style_transfer/style_transfer.py

```diff
@@ -634,6 +661,154 @@ def _raise_error_if_not_training_sframe(dataset, context_column):
     if dataset[context_column].dtype != _tc.Image:
         raise _ToolkitError("Context Image column must contain images")
 
+class StyleTransfer_beta(_Model):
+    """
+    A trained model using C++ implementation that is ready to use for classification or export to
+    CoreML.
+
+    This model should not be constructed directly.
+    """
+    _CPP_ACTIVITY_CLASSIFIER_VERSION = 1
+
+    def __init__(self, model_proxy=None, name=None):
+        self.__proxy__ = model_proxy
+        self.__name__ = name
+
+    @classmethod
+    def _native_name(cls):
```

Comment:
In custom model import we need a name here. Perhaps "style_transfer_beta" or the same name as above?

Reply:
Please consult with what @fareeha did for OD and AC to make this work for both settings of the feature flag